WooCommerce/Classes/ViewRelated/Dashboard/MyStore/PeriodDataViewController.swift

```diff
@@ -368,8 +386,7 @@ extension PeriodDataViewController: IAxisValueFormatter {
                 // Do not show the "0" label on the Y axis
                 return ""
             } else {
-                yAxisMaximum = value.humanReadableString()
-                return CurrencyFormatter().formatCurrency(using: yAxisMaximum,
+                return CurrencyFormatter().formatCurrency(using: value.humanReadableString(),
```

Comment:
👏 🎉 

I'm glad you removed `yAxisMaximum = value.humanReadableString()`. It was very odd that a `stringForValue(_)` method **mutates** the state (`yAxisMaximum` prop). 😱